sample/ios/ReactNativeTests/ShopifyCheckoutSheetKitTests.swift

```diff
@@ -201,6 +201,29 @@ class ShopifyCheckoutSheetKitTests: XCTestCase {
         XCTAssertEqual(mock.eventBody as? String, testEventJSON)
     }
 
+    /// checkoutDidStart
+    func testCheckoutDidStartSendsEvent() {
+        let mock = mockSendEvent(eventName: "started")
+
+        mock.startObserving()
+
+        // Create a test JSON string matching the CheckoutStartedEvent structure
+        let testEventJSON = """
+        {
+            "cart": {
+                "id": "test-cart-id",
+                "token": "test-cart-token"
+            }
+        }
+        """
+
+        // Simulate the event by calling sendEvent directly with the JSON string
+        mock.sendEvent(withName: "started", body: testEventJSON)
+
+        XCTAssertTrue(mock.didSendEvent)
+        XCTAssertEqual(mock.eventBody as? String, testEventJSON)
+    }
```

Comment:
Looks like the `CheckoutOptions` argument was added in a previous PR, but we don't have any tests for pass it.

The Android tests contain:
- testCanPresentCheckoutWithAuthenticationOptions()
- testCanPreloadCheckoutWithAuthenticationOptions()
- testCanPresentCheckoutWithNullOptions()
- testCanPreloadCheckoutWithNullOptions()

Want to add the same for iOS?